front-end/src/components/WorkoutsDropdowns.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { exerciseDefaults } from '../defaults/index';
+import Collapsible from 'react-collapsible';
+
+export default class WorkoutsDropdowns extends Component {
```

Comment:
This is a good way of thinking about this. Just another way of doing it so there is a lot less repeated code.

Instead of having 3 separate methods that do almost the same thing. You can have just one
```
renderExerciseName = (type) => {
    const exercises = this.state.exercises;
    const filteredExercises = exercises.filter(exercises => {
      return exercises.categoryName === type;
    });
    const filteredExerciseNames = filteredExercises.map(exerciseType => (
      <p>{exerciseType.exerciseName}</p>
    ));
    return filteredExerciseNames;
  };
```
Then all you have to do is pass in the type. After that instead of hard coding 4 different collapsibles. You can do something like this to make it more simple and even expandable.

```
return (
      <div className="workouts-dropdowns">
        {['Arms', 'Legs', 'Cardio', 'Abs' ].map(type => {
            <Collapsible trigger={type}>{this.renderExerciseName(type)}</Collapsible>
        })}
      </div>
    );
```

Just want you to see a way of doing it without repeating, plus you can add on things by adding to that list if users add more types

Reply:
Let me know what you think. Not manditory

Reply:
I sent in my pull yesterday thinking to myself, "Man, I hope Chris can show me a more DRY way of doing this." The callback hit me when I was half-asleep laying in bed lol.

Reply:
I'll work on re-factoring and re-request once I'm done.

Reply:
I cherish these comments haha